Julia/Julia.jl

```diff
@@ -0,0 +1,16 @@
+longest = String[]
+max_len = 0
+invalid(word) = occursin(r"[gkmqvwxzioGKMQVWXZIO]", word)
```

Comment:
Couldn't this be written as `r"[gkmqvwxzio]"i`?

Reply:
Yup, definitely missed out on that one.